core/src/main/scala/org/bitcoins/core/psbt/PSBT.scala

```diff
@@ -257,21 +255,18 @@ case class PSBT(
       s"index must be less than the number of output maps present in the psbt, $index >= ${outputMaps.size}")
     require(!isFinalized, "Cannot update a PSBT that is finalized")
 
-    val previousElements = outputMaps(index).elements
+    val map = outputMaps(index)
```

Comment:
lets call this `outputMap` and the above `inputMap`, scala lets your overshadow names but it can lead to confusion when reading and changing code